activity.py

```diff
@@ -69,12 +69,29 @@ class ActivityType(sequence_ordered(), DeactivableMixin, ModelSQL, ModelView):
 
     @classmethod
     def __register__(cls, module_name):
+        cursor = Transaction().connection.cursor()
         sql_table = cls.__table__()
+        table = cls.__table_handler__(module_name)
+
+        has_description = table.column_exist('default_duration')
```

Comment:
default_duration? I think you meant 'default_description?

Reply:
FIX 4746785